core/templates/dev/head/css/oppia.css

```diff
@@ -1287,31 +1301,30 @@ textarea {
   color: #fff;
 }
 
-li.sort-explorations-select {
-  display: block;
-  margin-left: auto;
+li.sort-explorations-select, span.sort-explorations-select {
```

Comment:
Just checking, do we still have li.sort-explorations-select? You've removed it in the rest of the CSS, so I wanted to make sure.